### PR TITLE
Update README.md

### LINKED ISSUE #110: Remove footnote
In the README.md, there's a footnote:
> Calendars marked with an asterisk (*) have not yet been released.

But no line was marked, probably meaning that all calendars are already released, right?

Should we add more calendars or remove the footnote?

### PR DESCRIPTION
**Pull request by willianpaixao**
Close #110

* Adding TravisCI build badge
* Remove unused footnote (#110)
* Include Installation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # trading_calendars
+
+[![Build Status](https://travis-ci.org/quantopian/trading_calendars.svg?branch=master)](https://travis-ci.org/quantopian/trading_calendars)
```
Comment (jmccorriston) [inline README.md]: I'm not familiar with the convention of including this in a README, is this standard? (disclaimer, I'm not an engineer :) )

File: README.md [hunk 2]
```diff
@@ -52,12 +55,17 @@ A Python library of exchange calendars meant to be used with [Zipline](https://g
...
 | Pakistan Stock Exchange         | XKAR     | Pakistan       | 1.11          | https://www.psx.com.pk/                                      |
 
-Calendars marked with an asterisk (*) have not yet been released.
+> Note that exchange calendars are defined by their [ISO-10383](https://www.iso20022.org/10383/iso-10383-market-identifier-codes) market identifier code.
 
-Note that exchange calendars are defined by their [ISO-10383 market identifier code](https://www.iso20022.org/10383/iso-10383-market-identifier-codes).
+## Installation
+
+``` bash
+$ pip install trading-calendars
```
Comment (jmccorriston) [inline README.md]: Nice addition, thank you!

File: README.md [hunk 3]
```diff
@@ -52,12 +55,17 @@ A Python library of exchange calendars meant to be used with [Zipline](https://g
 | Buenos Aires Stock Exchange     | XBUE     | Argentina      | 1.11          | https://www.bcba.sba.com.ar/                                 |
 | Pakistan Stock Exchange         | XKAR     | Pakistan       | 1.11          | https://www.psx.com.pk/                                      |
 
-Calendars marked with an asterisk (*) have not yet been released.
```
Comment (jmccorriston) [inline README.md]: We don't have any calendars that we're planning to add at this point, so you're right that we should remove the footnote. Thanks!

File: README.md (hunk 1 above)
Comment (quantophred) [reply README.md]: Yep, this is the Markdown syntax for a badge like the ones at the top of the Zipline README: https://github.com/quantopian/zipline/blob/master/README.rst

**Comment by gerrymanoim** [comment]
@willianpaixao Looks great, thanks!

**Pull request merged**